src/wazuh_db/helpers/wdb_agents_helpers.h

```diff
@@ -31,8 +31,8 @@ typedef enum agents_db_access {
  * @return Returns cJSON object with the sys_osinfo table information. Null in case of error.
  *                 The cJSON object must be freed by the caller.
  */
-cJSON* wdb_agents_sys_osinfo_get(int id,
-                                 int *sock);
+cJSON* wdb_get_sys_osinfo(int id,
```

Comment:
I think we should name this method `wdb_get_agent_sys_osinfo()`. Not in reference to the _agents_ database. Only to better explain that isn´t an osinfo of the manager or the host of the caller process.

Reply:
Updated!